### PR TITLE
Turned-on Security Baton goes: Shock.

### PR DESCRIPTION
**Pull request by darkmanx37**
## About The Pull Request

solves https://github.com/tgstation/tgstation/issues/62007 while making the code ever so slightly more efficient.

## Why It's Good For The Game

Now, a turned on baton will shock you instead of beating like any other. It's very useless but important difference.

## Changelog
:cl:
fix: Turned-off batons *beat* people, turned-on batons - shock them.
/:cl:

### CODE REVIEW

File: code/game/objects/items/melee/baton.dm [hunk 1]
```diff
@@ -559,9 +566,14 @@
...
 
 /obj/item/melee/baton/security/proc/scramble_mode()
+<<<<<<< Updated upstream
 	active = !active
 	playsound(src, "sparks", 75, TRUE, -1)
 	update_appearance()
+=======
+	set_active(!active)
+	playsound(src, "sparks", 75, TRUE, -1)
+>>>>>>> Stashed changes
```
Comment (lewcc) [inline code/game/objects/items/melee/baton.dm:576]: You still have leftovers from a failed merge/stash that are breaking things.

Comment (darkmanx37) [reply code/game/objects/items/melee/baton.dm:576]: where exacly and how to fix it?

Comment (darkmanx37) [reply code/game/objects/items/melee/baton.dm:576]: nevermind I am pretty dense

**Comment by jughu** [comment]
stunbeats you

File: code/game/objects/items/melee/baton.dm [hunk 2]
```diff
@@ -474,8 +482,7 @@
 	. = cell.use(deducted_charge)
 	if(active && cell.charge < cell_hit_cost)
 		//we're below minimum, turn off
-		active = FALSE
-		update_appearance()
+		set_active(FALSE)
 		playsound(src, "sparks", 75, TRUE, -1)
```
Comment (ShizCalev) [inline code/game/objects/items/melee/baton.dm:486]: should be handled by set_active

File: code/game/objects/items/melee/baton.dm [hunk 3]
```diff
@@ -559,9 +566,8 @@
 			addtimer(CALLBACK(src, .proc/scramble_mode), scramble_time*loops * (1 SECONDS))
 
 /obj/item/melee/baton/security/proc/scramble_mode()
-	active = !active
+	set_active(!active)
 	playsound(src, "sparks", 75, TRUE, -1)
```
Comment (ShizCalev) [inline code/game/objects/items/melee/baton.dm:570]: .

File: code/game/objects/items/melee/baton.dm [hunk 4]
```diff
@@ -454,18 +453,27 @@
 
 /obj/item/melee/baton/security/attack_self(mob/user)
 	if(cell?.charge >= cell_hit_cost)
-		active = !active
+		set_active(!active)
 		balloon_alert(user, "turned [active ? "on" : "off"]")
 		playsound(src, "sparks", 75, TRUE, -1)
```
Comment (ShizCalev) [inline code/game/objects/items/melee/baton.dm:458]: .

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**